packages/experience/src/Providers/UserInteractionContextProvider/UserInteractionContext.tsx

```diff
@@ -52,6 +74,8 @@ export default createContext<UserInteractionContextType>({
   setSsoEmail: noop,
   setSsoConnectors: noop,
   identifierInputValue: undefined,
+  // eslint-disable-next-line unicorn/no-useless-undefined
+  getIdentifierInputValueByTypes: () => undefined,
```

Comment:
use `noop`

Reply:
Per offline discussion, keep the status quo, since the `noop` can't used in the typed function.
![image](https://github.com/user-attachments/assets/182d38e9-837b-4785-a0bb-ba8f518ef8ea)